### DISCUSSION TITLE
I am not able to run code against a contract in production

### DISCUSSION THREAD

**Discussion by jonbulmer** [rainbow-me/rainbowkit#1542] [Feedback]
HI All

I have been working with Rainbow Kit and the Wagmi CLI on my new site using react. I have connected to the Time Dividend contract on my development machine and populated data from the contract read function using the Wagmi code within the APIs. Thank you for that.

My basic code is here.
https://github.com/jonbulmer/BasicWagmiSite

My issue is when I run the npm run build to get the code ready for the react code for production. Then port that code to a linux ubuntu machine running NGnix. The contract code stops working and I am not getting any return from the function calls. I am not able to find any help in the chrome inspection tools. 

I am now out of ideas and will start to look for other solutions to interact with block chain contract.

I have recorded a short video on you tube to try and further explain the issue.

https://youtu.be/DpS2M5TOVFM

Thank you for your help

Jon Bulmer

**Comment by raffle-x-dev** [comment]
Wow

**Comment by 0xmkh** [comment]
@jonbulmer It seems like when running your build locally it worked for me.

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/8b442a70-d771-49fc-a183-c6f33ac97242)

I see that you've got a weird console error. Can you check and see if it's something wrong with metamask extension ? [Related issue](https://stackoverflow.com/questions/77161596/lockdown-failed-typeerror-at-intrinsics-object-groupby-expected-boolean-not-fu)

Also can you delete your `node_modules` and `pnpm-lock.yaml` and try to run `pnpm install` again ?

**Reply by 0xmkh** [reply]
It might be also releated to this wagmi issue if you're using `create-react-app`: https://github.com/wevm/wagmi/issues/2916

**Comment by jonbulmer** [comment]
Thank you for the reply. I have moved on to other issues but I will go back
to this and try out your suggestions. I will let you know how it goes.
